euphonic/force_constants.py

```diff
@@ -453,19 +453,23 @@ def _calculate_phonons_at_qpts(
         cext_err_msg = (f'Euphonic\'s C extension couldn\'t be imported '
                         f'from {euphonic_path}, it may not have been '
                         f'installed.')
-        try:
-            if use_c is not False:
-                try:
-                    import euphonic._euphonic as euphonic_c
-                except ImportError:
-                    if use_c is None:
-                        warnings.warn((
-                            cext_err_msg
-                            + ' Falling back to pure Python calculation.'),
-                            stacklevel=3)
-                    raise
-            else:
-                raise ImportError
+
+        # Check if C extension can be used and handle appropriately
+        use_c_status = False

```

Comment:
Good call. Using a flag like this isn't as "pretty" as the other approaches we considered, but in this case it's clearly the most readable.